package.json

```diff
@@ -35,6 +38,8 @@
   },
   "dependencies": {
     "@types/google-apps-script": "^1.0.55",
+    "ci": "^2.2.0",
```

Comment:
Why do we need this?

Reply:
I had this to smoothly resolve package versions in different branches. I suppose I can move it to dev dependencies.